### ISSUE TITLE
Jira Api Support

### ISSUE THREAD

**Issue by latortuga71** [GhostManager/Ghostwriter#413]
Hello! im hoping to add jira api support on par with slack etc.

**Is your feature request related to a problem? Please describe.**

Jira API support to create a jira project when a ghost writer project is created.
When findings are linked to a report. create issue in corresponding jira project.
If finding is unlinked delete corresponding jira issue in project.

**Describe the solution you'd like**
Support for the jira api. I have a fork that is basically a MVP [(https://github.com/latortuga71/Ghostwriter)](https://github.com/latortuga71/Ghostwriter) -> jira_feature branch

Please let me know what you guys think.

**Comment by ColonelThirtyTwo** [comment]
I'm not sure if adding support for Jira specifically is a good idea - there are potentially many other different systems that one would want to automatically create when a project is created.

You might be able to do this today by having a service polling the GraphQL API and checking for new projects, though that requires a service to do so. A cleaner solution may be a notification or hooking system for project creation in Ghostwriter, that could then run a script or push to an endpoint when certain things happen.

**Comment by chrismaddalena** [comment]
I agree. Another option would be using the subscriptions feature of GraphQL to receive live updates on project creation:

https://hasura.io/docs/2.0/subscriptions/postgres/streaming/index/
